src/content/1.7/development/coding-standards/_index.md

```diff
@@ -69,23 +69,40 @@ class MyClass
 
 ### Deprecations
 
-Following [Symfony conventions](https://symfony.com/doc/3.4/contributing/code/conventions.html#deprecating-code), method and class deprecations in PrestaShop must be noted by adding the appropriate Phpdoc as well as a deprecation error:
+Following [Symfony conventions](https://symfony.com/doc/4.4/contributing/code/conventions.html#deprecating-code), method and class deprecations in PrestaShop must be noted by adding the appropriate Phpdoc as well as a deprecation error:
 
 ```php
 <?php
+
+namespace PrestaShop\Awesome\Path;
+
+@trigger_error(
+    sprintf(
+        '%s is deprecated since version 1.7.8.0 and will be removed in the next major version.',
+        MyClass::class
+    ),
+    E_USER_DEPRECATED
+);
+
 /**
- * @deprecated Since 1.7.6.0, use AnotherClass::someNewMethod() instead.
+ * @deprecated Since 1.7.8.0 and will be removed in the next major.
  */
-public function someOldMethod()
+class MyClass
 {
-    @trigger_error(
-        sprintf(
-            '%s is deprecated since version 1.7.6.0. Use %s instead.',
-            __METHOD__,
-            AnotherClass::class . '::someNewMethod()'
-        ),
-        E_USER_DEPRECATED
-    );
+    /**
+     * @deprecated Since 1.7.6.0, use AnotherClass::someNewMethod() instead.
+     */
+    public function someOldMethod()
+    {
+        @trigger_error(
+            sprintf(
+                '%s is deprecated since version 1.7.6.0. Use %s instead.',
+                __METHOD__,
+                AnotherClass::class . '::someNewMethod()'
+            ),
+            E_USER_DEPRECATED
+        );
```

Comment:
What do you want to depreciate concretely? The whole class or just a method?
If the whole class is deprecated, is it still necessary to specify in the methods?

Reply:
The method is deprecated since 1.7.6.0 and the class since 1.7.8.0, it's an example following the Symfony coding standard :smile_cat: